TEMPLATE/test/test_db_seed.py

```diff
@@ -0,0 +1,18 @@
+from TEMPLATE.commands import drop_all_tables
+import flask_migrate
+from TEMPLATE.db.fixtures import seed_db
+from TEMPLATE.db import db
+
+
+def test_db_init_seed(app):
+    """Try initializing and seeding development database."""
+    drop_all_tables(app=app)
+    db.create_all(app=app)
+    seed_db()
+    drop_all_tables(app=app)
```

Comment:
Is the last `drop_all_tables(app=app)` a mistake? Looks wrong to me

Reply:
it's testing that drop_all_tables() works

Reply:
Ooooh, I didn't notice that this a test 🤦‍♂